src/Nix/String/Coerce.hs

```diff
@@ -38,6 +38,15 @@ data CopyToStoreMode
   -- ^ Add paths to the store as they are encountered
   deriving (Eq,Ord,Enum,Bounded)
 
+newtype CoerseAsyncException a
```

Comment:
`CoerseAsyncException` → `CoerceAsyncException`

Reply:
Thank you.